cfpb/consumerfinance.gov
# Expandables: merge padded expandable modifier into default

Bring in changes from https://github.com/cfpb/design-system/pull/1924

## Changes

- Update to design-system 0.38.0, eslint-plugin-react, snyk
- Remove unneeded `o-expandable__padded` classes.


## How to test this PR

1. `yarn build` and expandables across the site should be the same as before.

iregs (resize to mobile for expandable)

http://localhost:8000/rules-policy/regulations/1002/1/

TDP
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/


## Notes and todos

- iregs secondary nav is very customized and non-conforming. There are TODOs to get it untangled.
